scripts/Refinery.js

```diff
@@ -160,120 +162,146 @@ async function isTargetRefineryCheck(tokenAddress, amount) {
     targetFundMangerContract.address
   );
 
-  if (isTokenFoundryAsset == false && Number(targetFundMangerLiquidity) > Number(amount)) {
-    return true; 
-  }else {
-      return false;
-    }
+  if (
+    isTokenFoundryAsset == false &&
+    Number(targetFundMangerLiquidity) > Number(amount)
+  ) {
+    return true;
+  } else {

```

Comment:
I think else { could be removed and return false directly for code optimization.